index.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 generated: 2019-06-24T21:00:00+00:00
 projects:
-  java-microprofile:
+  java-microprofile-mct:
```

Comment:
Why do we need to add -mct?

Reply:
The `mct` should be removed and version updated for the same stack.